lib/driver.js

```diff
@@ -723,6 +717,12 @@ class XCUITestDriver extends BaseDriver {
         const devices = await getConnectedDevices();
         log.debug(`Available devices: ${devices.join(', ')}`);
         if (devices.indexOf(this.opts.udid) === -1) {
```

Comment:
can we use `includes` instead?

Reply:
Sure, but  I'd rather do it under another PR, since if we are changing this we might as well change the dozens of other places where this pattern is happening.